apps/cli-daemon/src/app/generators/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { readdirSync, readFileSync } from 'fs';
+import { join } from 'path';
+
+import { GeneratorDefinition, Schema } from './generators.interface';
+
+const capitalize = (s: string): string => s[0].toUpperCase() + s.slice(1);
+const formatSchemaName = (name: string): string => {
+  const splitName = name.split('-');
+  if (splitName.length > 1) {
+    return (
+      capitalize(splitName[0]) + capitalize(splitName[splitName.length - 1])
+    );
+  }
+  return capitalize(splitName[0]);
+};
+export const getGeneratorDefinition = (
+  generatorName: string,
+  path: string
+): GeneratorDefinition | null => {
+  try {
+    const schema: Schema = JSON.parse(
+      readFileSync(`${path}/${generatorName}/schema.json`, 'utf-8')
+    );
+    const displayName = formatSchemaName(generatorName);
+    return {
+      displayName,
+      originalName: generatorName,
+      description: schema.description,
+    };
+  } catch (e) {
+    console.log(`There is no schema for ${generatorName}`);
+    return null;
+  }
+};
+export const getGeneratorNames = (path: string): string[] =>
+  readdirSync(path)
+    .map((name) => join(path, name))
+    .map((directoryName) => directoryName.split(path + '/')[1]);
+
+export const convertKeyToArgument = (key: string): string => {
```

Comment:
can you explain this function and add comments with an example? I can't really understand what it does

Reply:
added comments, but main idea is to take a param for example `inlineStyle` and to turn it to an argument `--inline-style`

Reply:
thanks now it's understandable, better to write comments on the function itself so it can be seen when you inspect the function/generate docs

/**
* comments...
*/
function()